Packs/Devo/Integrations/Devo_v2/Devo_v2.yml

```diff
@@ -159,9 +152,6 @@ script:
           description: Start datetime for the specified query. This argument supports natural language (e.g., 2 day, 3 week), Unix timestamps, Python datetime objects, and string datetimes
         - name: to
           description: End datetime for specified query. If provided must be in same format as "from" argument. This argument is ignored in a date range.
-        - name: limit
```

Comment:
I am deleting this argument because it was incorrectly implemented, and we deleted any usages to it in the previous PR. We can restore it, but this argument does nothing at the moment.